### DISCUSSION TITLE
Could you please inform us on how you have set up your CALERIE dataset?

### DISCUSSION THREAD

**Discussion by manojvaithilingam** [dayoonkwon/BioAge#34] [General]
We were fortunate enough to now have access to the original dataset but wanted to replicate your findings. However, the dataset includes multiple CSV files. We were therefore wondering how you structured your df. Thanks so much!

**Comment by dayoonkwon** [comment]
The files can be merged using the identifiers deidnum for individuals and visits for time of measurement.
